go/filemanager/pfsmodules/command.go

```diff
@@ -0,0 +1,78 @@
+package pfsmodules
+
+import (
+	"errors"
+	"io"
+	"net/url"
+	"strings"
+
+	log "github.com/golang/glog"
+)
+
+const (
+	// DefaultMultiPartBoundary is the default multipart form boudary.
+	DefaultMultiPartBoundary = "8d7b0e5709d756e21e971ff4d9ac3b20"
+
+	// MaxJSONRequestSize is the max body size when server receives a request.
+	MaxJSONRequestSize = 2048
+)
+
+// Command is a interface of all commands.
+type Command interface {
+	ToURLParam() url.Values
```

Comment:
Better to add comments for each interface usages? like
// ToURLParam convert the current pfs command to a url.Values struct which will be used by pfs server and client.